avm/res/communication/email-service/main.bicep

```diff
@@ -0,0 +1,181 @@
+metadata name = 'Email Services'
+metadata description = 'This module deploys an Email Service'
+metadata owner = 'Azure/module-maintainers'
+
+@minLength(1)
+@maxLength(63)
+@description('Required. Name of the email service to create.')
+param name string
+
+@description('Optional. Location for all Resources.')
```

Comment:
Hey @krbar, given what you specified in the tests - should the default location always be 'global'?

Reply:
Thanks, I changed it to "global"